### ISSUE TITLE
Assessment instance issues with Windows 11 high contrast themes

### ISSUE THREAD

**Issue by jalkire** [microsoft/accessibility-insights-web#5098]
## Describe the bug
When using certain high contrast themes in Windows 11, the undo and visibility toggle buttons on assessment instances from assisted tests (and the yet to be released tab stops instances) are nearly invisible. In Chrome, the text of selected instances is also unreadable.

**To Reproduce**

1. Install the extension
2. Enable High Contrast in the Windows settings. The Night sky theme makes the issue very apparent in Windows 11.
3. Run an assessment against a page that will have some assisted assessment requirement instances (such as [the AU demo site](http://www.washington.edu/accesscomputing/AU/before.html))
4. Navigate to a requirement with instances (such as Native widgets -> Instructions)
5. Observe a lack of undo and visibility toggle buttons

## Expected behavior

All buttons should be visible

## Screenshots
Using the Night sky contrast theme in Win 11

Edge: 
![screenshot showing instances with the issue in edge](https://user-images.githubusercontent.com/4615491/150613712-1de99a77-4ef8-4531-b9c6-c4a6111b5788.png)
Chrome:
![screenshot showing instances with the issue in chrome](https://user-images.githubusercontent.com/4615491/150613727-c4ddfd23-1e0d-463b-95e4-379e500f217c.png)

## Context (please complete the following information)

- OS Name & Version: Windows 11 22000.434
- AI-Web Version & Environment: 2.29.0 prod
- Browser Version: Edge Version 97.0.1072.69 (Official build) (64-bit) and Chrome Version 97.0.4692.99 (Official Build) (64-bit)

## Additional context
Found during release validation for 2.30.0

**Comment by jalkire** [comment]
Confirmed fixed in Edge with insider 2.30.1. However, the selected instance row text color issue remains in Chrome

**Comment by brocktaylor7** [comment]
I am not able to replicate the issue in chrome, it is working as I expected for me:
![image](https://user-images.githubusercontent.com/13286385/151851578-06a2053d-d8e7-4101-bb38-6760eb08e296.png)

**Comment by jalkire** [comment]
Looks like I didn't give repro steps for the Chrome text color issue--it occurs when navigating the instances list with keyboard, and only after moving from one instance to another:
![gif showing more detailed repro of bad behavior](https://user-images.githubusercontent.com/4615491/151853233-e0dc62c2-0958-408e-977c-2db67e6e3247.gif)
